File: rahul.c
#include<stdio.h>
int main()
{
	int i,j,n,fact;
	
	printf("enter the value of n");
	scanf("%d",&n);

	for(i=1;i<=n;i++)
	{
		fact=0;
		for(j=1;j<=i;j++)
		{
			if(i%j==0)
			{
				fact++;
			}


		}

		if(fact==2)
		{
			printf("%d\n",i);
		}
	}
}

File: rahul.php
<?php
$j=1;
//echo "rahul please enter the number ==";
//$a=fgets(STDIN);

for($i=1;$i<=$20;$i++)
{
	$fact=0;
	while($j<=$i)
	{
		if($i%$j==0)
		{ $fact++;
		}
	}

	if($fact==2)
	{
		echo $i;
	}
}
?>
